provider/pkg/gen/nodejs-templates/helm/v2/helm.ts

```diff
@@ -203,7 +204,27 @@ export class Chart extends yaml.CollectionComponentResource {
                 const namespaceArg = cfg.namespace
                     ? `--namespace ${shell.quote([cfg.namespace])}`
                     : "";
-                let cmd = `helm template ${chart} --name-template ${release} --values ${defaultValues} --values ${values} ${apiVersionsArgs} ${namespaceArg}`;
+
+                // Check the helm version - v2 or v3
+                let helmVerCmd = `helm version --short || true`;
+                const helmVer = execSync(
+                    helmVerCmd,
+                    {
+                        stdio: ['pipe', 'pipe', 'ignore'], // Suppress tiller version error
+                    },
+                ).toString();
+
+                // Helm v2 returns version like this:
+                // Client: v2.16.7+g5f2584f
+                // Helm v3 returns a version like this:
+                // v3.1.2+gd878d4d
+                // --include-crds is available in helm v3.1+ so check for a regex matching that version
+                let r = RegExp('^v3\.[1-9]');
+                if (r.test(helmVer)) {
+                    cmd = `helm template ${chart} --include-crds --name-template ${release} --values ${defaultValues} --values ${values} ${apiVersionsArgs} ${namespaceArg}`;
```

Comment:
Sorry, one more suggestion. How about appending the `--include-crds` arg in this case rather than having both cases here explicitly?

Something like
```ts
cmd = `helm template ${chart} --name-template ${release} --values ${defaultValues} --values ${values} ${apiVersionsArgs} ${namespaceArg}`;
if (RegExp('^v3\.[1-9]').test(helmVer) {
    cmd += ` --include-crds`
}
```

Reply:
Good idea, I'll rework it.